gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Cluster.java

```diff
@@ -328,10 +328,10 @@ public int getMaxWaitForClose() {
     }
 
     /**
-     * Gets the maximum size in bytes of any request sent to the server.
+     * Gets the maximum size in bytes of any request received from the server.
      */
-    public int getMaxContentLength() {
-        return manager.connectionPoolSettings.maxContentLength;
+    public double getMaxResponseContentLength() {
```

Comment:
Why not `Long`?

Reply:
Hello @vkagamlyk thanks for taking a look at my PR!
It was suggested by @kenhuuu that using double would provide more flexibility to users that need a maximum that is larger than the Long.MAX_VALUE.

Reply:
Long.MAX_VALUE is 9 223 372 terabytes. Do users really need that kind of range?

Reply:
Discussed with @kenhuuu and will change to Long.

Reply:
Changed to long